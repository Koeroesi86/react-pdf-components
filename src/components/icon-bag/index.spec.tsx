import React from "react";
import { renderToString } from "@react-pdf/renderer";
import { Wrapper } from "@test-helpers";
import IconBag from "./index";

describe("<IconBag/>", () => {
  it("should render", async () => {
    const result = await renderToString(
      <Wrapper>
        <IconBag width={12} height={12} color="#000"/>
      </Wrapper>
    );
    expect(result).toMatchSnapshot();
  });
});
