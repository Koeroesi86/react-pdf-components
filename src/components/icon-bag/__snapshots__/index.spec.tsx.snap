// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<IconBag/> should render 1`] = `
"%PDF-1.3
%����
7 0 obj
<<
/Type /Page
/Parent 1 0 R
/MediaBox [0 0 595.280029 841.890015]
/Contents 5 0 R
/Resources 6 0 R
>>
endobj
6 0 obj
<<
/ProcSet [/PDF /Text /ImageB /ImageC /ImageI]
>>
endobj
5 0 obj
<<
/Length 707
>>
stream
1 0 0 -1 0 841.890015 cm
q
q
1 0 0 1 0 0 cm
q
q
0 3 m
12 3 l
12 3 12 3 12 3 c
12 15 l
12 15 12 15 12 15 c
0 15 l
0 15 0 15 0 15 c
0 3 l
0 3 0 3 0 3 c
h
W n
1 0 0 1 0 3 cm
0.023438 0 0 0.023438 0 0 cm
1 0 0 1 0 0 cm
q
/DeviceRGB cs
0 0 0 scn
320 336 m
320 344.84 312.84 352 304 352 c
208 352 l
199.16 352 192 344.84 192 336 c
192 288 l
0 288 l
0 432 l
0 457.6 22.4 480 48 480 c
464 480 l
489.6 480 512 457.6 512 432 c
512 288 l
320 288 l
320 336 l
h
464 128 m
384 128 l
384 80 l
384 54.4 361.6 32 336 32 c
176 32 l
150.4 32 128 54.4 128 80 c
128 128 l
48 128 l
22.4 128 0 150.4 0 176 c
0 256 l
512 256 l
512 176 l
512 150.4 489.6 128 464 128 c
h
320 128 m
192 128 l
192 96 l
320 96 l
320 128 l
h
f
Q
Q
Q
Q
Q

endstream
endobj
9 0 obj
(react-pdf)
endobj
10 0 obj
(react-pdf)
endobj
11 0 obj
(D:20210414183756Z)
endobj
8 0 obj
<<
/Producer 9 0 R
/Creator 10 0 R
/CreationDate 11 0 R
>>
endobj
3 0 obj
<<
/Type /Catalog
/Pages 1 0 R
/Names 2 0 R
/ViewerPreferences 4 0 R
>>
endobj
1 0 obj
<<
/Type /Pages
/Count 1
/Kids [7 0 R]
>>
endobj
2 0 obj
<<
/Dests <<
  /Names [
]
>>
>>
endobj
4 0 obj
<<
/DisplayDocTitle true
>>
endobj
xref
0 12
0000000000 65535 f 
0000001210 00000 n 
0000001267 00000 n 
0000001123 00000 n 
0000001314 00000 n 
0000000200 00000 n 
0000000133 00000 n 
0000000015 00000 n 
0000001049 00000 n 
0000000958 00000 n 
0000000985 00000 n 
0000001013 00000 n 
trailer
<<
/Size 12
/Root 3 0 R
/Info 8 0 R
>>
startxref
1357
%%EOF
"
`;
